lib/abstract_format.ml

```diff
@@ -954,8 +943,12 @@ and fun_type_of_sf sf : (type_t, err_t) Result.t =
   | Sf.Tuple (4, [Sf.Atom "type";
                   Sf.Integer line;
                   Sf.Atom "fun";
-                  Sf.List [sf_params; sf_ret]]) ->
-     let%bind params = sf_params |> type_of_sf |> track ~loc:[%here] in
+                  Sf.List [Sf.Tuple (4, [Sf.Atom "type";
+                                         Sf.Integer _;
```

Comment:
Please do not discard information. Changing the definition of `TyFun of line_t * type_t list * type_t` to `TyFun of line_t * line_t * type_t list * type_t` is a better solution, I think.

Reply:
I fixed to retain line information.